salesforce-marketing-cloud/3.1/modules/ROOT/pages/salesforce-mktg-connector-reference.adoc

```diff
@@ -0,0 +1,840 @@
+
+* <<Config>>
+
+[[NewObjectListener]]
+== On new object
+`<sfdc-marketing-cloud:new-object-listener>`
+
+
+=== Parameters
+
+[%header%autowidth.spread]
+|===
+| Name | Type | Description | Default Value | Required
+| Configuration | String | The name of the configuration to use. | | x
+| Since a| String |  Specify a date in the yyyy-MM-dd HH:mm:ss format (that is, 2017-03-17 16:30:40) to retrieve the selected objects. |  |
+| Amount a| Number |  Specifies the amount of time, in the unit defined in Read timeout time unit, that the consumer waits for a response before it times out. |  |
+| Time unit a| Enumeration, one of:
+
+** NANOSECONDS
+** MICROSECONDS
+** MILLISECONDS
+** SECONDS
+** MINUTES
+** HOURS
+** DAYS |  A time unit that qualifies the Read timeout. |  |
+| Object type a| String |  |  | x
+| Config Ref a| ConfigurationProvider |  The name of the configuration to use to execute this component. |  | x
+| Primary Node Only a| Boolean |  Whether this source should only be executed on the primary node when running in Cluster. |  |
+| Scheduling Strategy a| scheduling-strategy |  Configures the scheduler that triggers the polling |  | x
+| Redelivery Policy a| <<RedeliveryPolicy>> |  Defines a policy for processing the redelivery of the same message. |  |
+| Reconnection Strategy a| * <<reconnect>>
+* <<reconnect-forever>> |  A retry strategy in case of connectivity errors. |  |
+|===
+
+=== Output
+
+[%autowidth.spread]
+|===
+|Type |Object
+| Attributes Type a| Any
+|===
+
+=== For Configurations
+
+* <<Config>>
+
+
+== Types
+[[ProxyConfiguration]]
+=== Proxy Configuration
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Host a| String | Host where the proxy requests is sent. |  | x
+| Port a| Number | Port where the proxy requests is sent. |  | x
+| Username a| String | The username to authenticate against the proxy. |  |
+| Password a| String | The password to authenticate against the proxy. |  |
+| Non Proxy Hosts a| Array of String | A list of hosts against which the proxy should not be used. |  |
+| Ntlm Domain a| String | The domain to authenticate against the proxy. |  |
+|===
+
+[[Tls]]
+=== Tls
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Enabled Protocols a| String | A comma-separated list of protocols enabled for this context. |  |
+| Enabled Cipher Suites a| String | A comma-separated list of cipher suites enabled for this context. |  |
+| Trust Store a| <<TrustStore>> |  |  |
+| Key Store a| <<KeyStore>> |  |  |
+| Revocation Check a| * <<StandardRevocationCheck>>
+* <<CustomOcspResponder>>
+* <<CrlFile>> |  |  |
+|===
+
+[[TrustStore]]
+=== Trust Store
+
+[%header,cols="20s,25a,30a,15a,10a"]
+|===
+| Field | Type | Description | Default Value | Required
+| Path a| String | The location (which will be resolved relative to the current classpath and file system, if possible) of the trust store. |  |
```

Comment:
S&R trust store with truststore. Same with key store. Unless it's referring to the UI. This is not important though so skip if you don't have time.